src/index.js

```diff
@@ -294,4 +294,35 @@ export default class Eik {
 			'Eik config was not loaded or "loadMaps" is "false" when calling .maps()',
 		);
 	}
+
+	/**
+	 * Function that generates and returns an import map script tag for use in an document head.
+	 *
+	 * Currently only a single import map is supported in the browser so we need to merge together import map objects into a single object
+	 * by using a JS Map object. Last in wins so order of import maps defined in eik.json is important if multiple maps share the same entries.
```

Comment:
The implementation detail with `Map` isn't that important to readers I think.

"Currently" makes me think that might change soonish, but this is a constraint from the spec, right? Is it likey to change?

```suggestion
	 * Only a single import map is allowed per HTML document.
	 * A key (ex. `react`) can only be defined once.
	 * If multiple import maps defined in `eik.json` use the same key the last one wins.
```

Reply:
There is talk at the spec level of changing this which is why I went with currently. It seems fine not to mention though.